avaframe/out3SimpPlot/plotSettings.py

```diff
@@ -8,11 +8,15 @@
 from matplotlib import cm
 import copy
 import matplotlib
+from matplotlib.image import NonUniformImage
 import cmocean
 import copy
 
+from avaframe.in3Utils import cfgUtils
 from avaframe.out3SimpPlot.makePalette import *
 
+cfgMain = cfgUtils.getGeneralConfig()
+continuousCmap = cfgMain['MAIN'].getboolean('continuousCmap')
 
```

Comment:
This is not a good idea:
apart from it not working on a clean install (see the test build of this PR (pytest) ), 
it is dangerous to read a config twice-> what if the user decides to dynamically change a setting variable in his main script without being aware that a module reads it again...
-> we should keep this the way it is at the moment: the main script reads the cfg and supplies this to the modules...